content/en/cosign/overview.md

```diff
@@ -15,98 +15,84 @@ features:
-{"Critical":{"Identity":{"docker-reference":""},"Image":{"Docker-manifest-digest":"sha256:87ef60f558bad79beea6425a3b28989f01dd417164150ab3baab98dcbf04def8"},"Type":"cosign container image signature"},"Optional":null}
+```
+cosign sign-blob --help
 ```
 
-## Kubernetes Integrations
+### Verifying a signed blob
 
-Cosign comes with a few built-in Kubernetes integrations: `Secret` generation, and a [policy webhook](/policy-controller/overview/) `policy-controller`.
-In addition to the `policy-controller`, Cosign is also compatible with and supported by other policy engines such as:
+To verify a signed blob, you need to provide three pieces of information:
+* The certificate, found in the bundle
+* The signature, also found in the bundle
+* The identity used in signing
 
-* [Conaisseur](https://github.com/sse-secure-systems/connaisseur#what-is-connaisseur)
-* [Kyverno](https://kyverno.io/docs/writing-policies/verify-images/)
-* [OPA Gatekeeper](https://github.com/sigstore/cosign-gatekeeper-provider)
+The following example verifies the signature on file.txt from user "name@example.com" issued by "accounts@example.com":
 
-To learn how to use Cosign with Kubernetes, review [Kubernetes](/cosign/kubernetes/).
+```
+$ cosign verify-blob file.txt --bundle txtbundle.bundle --certificate-identity=name@example.com 
+                              --certificate-oidc-issuer=https://accounts.example.com
+```
 
-## More Info
+To verify, Cosign queries the transparency log to compare the public key to what’s in Rekor, and checks the timestamp on the signature against the artifact’s entry in the transparency log. The signature is valid if its timestamp falls within the small window of time that the key pair and certificate issued by OpenID Connect were valid.
 
-Cosign can do much more than what is discussed here. Review more information on the commands by checking out the other sections of this site.
+### Working with containers
 
-### Other Formats
+Signing and verifying a container is similar to working with blobs.   The Cosign command to sign a container image is:
 
-Cosign is useful not only for containers and container-related artifacts; it can also be used for other file types. 
+```
+$ cosign sign <image URI>
+```
 
-To learn how to sign SBOMs, WASM modules, Tekton bundles and more, review [Signing Other Types](/cosign/other_types/). For basic blobs, review [Working with Blobs](/cosign/working_with_blobs/).
+This works the same as signing a blob, but the signature becomes attached to the container itself.
 
-### SCM Integration
+To verify a signed container image, use the following command:
 
-Cosign integrates natively with source code management (SCM) systems like GitHub and GitLab.
-You can use the official [GitHub Actions Cosign installer](https://github.com/marketplace/actions/cosign-installer) or use `cosign` to generate and work safely with [SCM secrets](/cosign/git_support/) with native API integration.
+```
+$ cosign verify <image URI>
```

Comment:
need to also include the identity and oidc-issuer flags here

Reply:
added.